Tests/Tactics/Sym.lean

```diff
@@ -7,15 +7,15 @@ import Proofs.«AES-GCM».GCMGmultV8Sym
 
 namespace GCMGmultV8Program
 
--- Test the behaviour of `sym1_n` when `h_err` and `h_sp` are not present
+-- Test the behaviour of `sym_n` when `h_err` and `h_sp` are not present
 /-- warning: declaration uses 'sorry' -/ -- `guard_msgs` silences the warning
 #guard_msgs in example (s0 sf : ArmState)
     (h_s0_program : s0.program = gcm_gmult_v8_program)
     (h_s0_pc : read_pc s0 = gcm_gmult_v8_program.min)
     (h_run : sf = run gcm_gmult_v8_program.length s0) :
     read_err sf = .None := by
   simp (config := {ground := true}) only [Option.some.injEq] at h_s0_pc h_run
-  sym1_n 1
+  sym_n 1
   · sorry
   · exact (sorry : CheckSPAlignment s0)
```

Comment:
For a subsequent PR: if `CheckSPAlignment s0` is not an original hyp., then don't generate this proof obligation.

Reply:
It's on my radar! (although a tiny bit more complicated than I thought: I removed the addition of the goals from `SymContext`, but apparently `intro_fetch_decode_lemmas` *also* introduces an `SPAlignment` goal if it can't find the hyp, I'll have to dig into that tactic)